_topic_maps/_topic_map.yml

```diff
@@ -3530,6 +3530,24 @@ Topics:
     File: kn-eventing-ref
   - Name: Functions commands
     File: kn-func-ref
+# Kafka
+- Name: Knative Kafka
+  Dir: knative_kafka
+  Topics:
+  - Name: About Knative Kafka
+    File: serverless-about-kafka
+  - Name: Installing Knative Kafka
+    File: serverless-install-kafka
+  - Name: Configuring Kafka security
+    File: serverless-kafka-security
+  - Name: Kafka source
+    File: serverless-kafka-source
+  - Name: Kafka broker
+    File: serverless-kafka-broker
+  - Name: Kafka channel
+    File: serverless-kafka-channel
+  - Name: Kafka sink
+    File: serverless-kafka-sink
```

Comment:
Hi @abrennan89 
The overall structure of Kafka docs looks good to me. 
I have a few suggestions:
1. We could have the Kafka Sink section right after the Kafka Source section (I have a similar reorg in mind for the core Eventing section.) The reason is, in the Source section we mention that we can send events from a source to a sink. So if we have the start and end of the Event delivery mechanism explained right at the start it would help the user understand the roles of all the in between actors like Brokers, channels etc.
2. After the channel section add a section for - Event delivery behavior patterns for Knative Kafka channels and brokers. Maybe change the name of [this section so that you can reuse it](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/serverless/develop#serverless-event-delivery-component-behaviors-kafka_serverless-event-delivery).

Reply:
WRT the placement of the Kafka section: I know we want to make them discoverable but yet,
I suggest right now we add it to the Develop section under Triggers. Even later, it would make sense to add it to the Eventing Section, so I envision it as a third level heading. 
Reasons: 
1. It is a Eventing functionality hence should be with the Eventing sections. 
2. Current position seems abrupt with little connection to the CLI section before it and the Develop section after it. 
3. I think adding it upfront right now for visibility and then moving it to third level could be jarring for predictability and findability.
4. Also if I am a new user I would first want to understand all the components of Eventing and then deep dive into the Kafka use case.